bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/IWorkbench.java

```diff
@@ -473,21 +472,6 @@ IWorkbenchPage showPerspective(String perspectiveId, IWorkbenchWindow window, IA
 	 */
 	IWorkbenchActivitySupport getActivitySupport();
 
-	/**
-	 * Returns an interface to manage commands at the workbench level.
-	 *
-	 * @return an interface to manage commands at the workbench level. Guaranteed
-	 *         not to be <code>null</code>.
-	 * @since 3.0
-	 * @deprecated Please use {@link IServiceLocator#getService(Class)} instead.
-	 *             This API is scheduled for deletion, see Bug 431177 for details
-	 * @see ICommandService
-	 * @see IHandlerService
-	 * @noreference IWorkbenchCommandSupport is scheduled for deletion.
-	 */
-	@Deprecated
```

Comment:
i am not using that API, so i dont have a problem. But maybe nobody read the javadoc "This API is scheduled for deletion, see Bug 431177 for details" it would be better to use
`@Deprecated(forRemoval=true)` in he future

Reply:
> @Deprecated(forRemoval=true)

That was added with Java 9, so probably at that time platform where still using java 8:
https://docs.oracle.com/javase/9/core/enhanced-deprecation1.htm

Reply:
@laeubi correct, most API deletions were before @deprecated(forRemoval=true) was availabler in platform. New API removals should use forRemoval and if someone has time and desire for that, he / she could also add this @deprecated(forRemoval=true) to the API already scheduled for deletion.  In general the current process of API deletion is relative complex which the update of the API removal document in the commons repo.